config/config.go

```diff
@@ -0,0 +1,6 @@
+package config
+
+const (
+	OperatorName      string = "aws-account-shredder"
```

Comment:
Do we need these? This is not an operator

Reply:
changed the variables.